manifests/zone.pp

```diff
@@ -51,6 +51,8 @@
 # @param zonefilepath
 # @param filename
 # @param forward
+# @param master_empty_forwarders_enable
+#   Enable empty forwarders option in master zone
```

Comment:
I think this is a bit brief. It doesn't explain that `forwarders` must be empty and `zonetype` set to `master`.

Reply:
Hello @ekohl,

Thank you for looking into this and for your feedback.

Description for `master_empty_forwarders_enable` parameter was updated as you suggested.